homeassistant/components/mazda/__init__.py

```diff
@@ -59,6 +63,53 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         _LOGGER.error("Error occurred during Mazda login request: %s", ex)
         raise ConfigEntryNotReady from ex
 
+    async def async_handle_service_call(service_call=None):
+        """Handle a service call."""
+        # Get device entry from device registry
+        dev_reg = device_registry.async_get(hass)
+        device_id = service_call.data.get("vehicle")
```

Comment:
Suggest to call this parameter `device_id` instead since it's a Home Assistant device_registry id.